packages/@aws-cdk/aws-s3-assets/README.md

```diff
@@ -117,6 +119,32 @@ new assets.Asset(this, 'BundledAsset', {
 Although optional, it's recommended to provide a local bundling method which can
 greatly improve performance.
 
+### Copying Directly From a Bundling Image
+
+You may set `copyFromImagePath` to `true` to instruct the bundler to copy an
+asset out of your bundling docker image. To use this feature, you should create
+your own bundling docker image from a `Dockerfile`:
```

Comment:
When would you absolutely recommend using this? I can see the benefit but I find it hard to determine and clearly explain which use case should use `docker run` and which should use `docker build + cp`?

For `aws-lambda-nodejs` do we agree that it's not a good option because it would mean copying the whole `node_modules` folder in the image?

Reply:
I would recommend using `docker build + cp` when the emitted asset consists of many small files. This is because copying thousands of files to the output volume in Docker for Windows is still slow with WSL 2. I'm not sure what is the case in macOS, but I suspect it's a similar situation.

My use case is a construct for a static site generator for my team. For one of my sites, the bundling/build results in nearly 40k files that I need to put on S3 behind CloudFront. Copying them from the bundler to the volume can take a minute longer than copying from the container with the docker command.

There's nothing stopping me from orchestrating the build outside of the bundler and creating the asset directly from those files. But, it would be more convenient if I could use the bundler for all the pre-existing docker-handling code.

Agreed that `aws-lambda-nodejs` may not benefit directly from this approach. Parcel is emitting a single file, right?